src/nexusGrades.js

```diff
@@ -87,6 +92,38 @@ fluid.nexus.writeDefaults.handleRequest = function (gradeName, request) {
     request.events.onSuccess.fire();
 };
 
+fluid.defaults("fluid.nexus.readComponent.handler", {
+    gradeNames: ["kettle.request.http"],
+    invokers: {
+        handleRequest: {
+            funcName: "fluid.nexus.readComponent.handleRequest",
+            args: ["{request}.req.params.path", "{request}", "{fluid.nexus}.nexusComponentRoot"]
+        }
+    }
+});
+
+// TODO: this API endpoint should really supply the potentia and model of a component,
+//       i.e. serialized material that could be used to reconstruct the component in its current state.
+//       This is not straightforward to provide under the current version of Infusion.
+/**
+ * Retrieve a serialized version of a component's "shell" at a path, consisting of its construction status, typeName, model, and id.
+ * @param {String} path the path to the component, can also be given as an array.
+ * @param {Object} request the kettle.request.http component that will mediate the response.
```

Comment:
Where there is a known grade these days we tend to try to be more precise in the docs and say, e.g. {kettle.request.http}